conda-requirements.txt

```diff
@@ -0,0 +1,12 @@
+argh
```

Comment:
In travis-setup.sh, I'm just trying to use conda to quickly install things that are available, and then add pip stuff later, i.e.,

```
conda install -y --file conda-requirements.txt --channel bioconda --channel conda-forge --channel anaconda
python setup.py install
pip install -r pip-test-requirements.txt
pip install -r pip-requirements.txt
```

I think I remember seeing a way of installing pip packages with conda so the separate files are not necessary but haven't played around with that.